README.md

```diff
@@ -7,9 +7,31 @@ Orgmode clone written in Lua for Neovim 0.5.
 Use your favourite package manager:
 
 * [vim-packager](https://github.com/kristijanhusak/vim-packager):
-  ```lua
-    packager.add('kristijanhusak/orgmode.nvim')
-  ```
+
+```lua
+packager.add('kristijanhusak/orgmode.nvim')
+```
+
+- [packer.nvim](https://github.com/wbthomason/packer.nvim)
+
+```lua
+use {
+    'kristijanhusak/orgmode.nvim',
```

Comment:
It's probably worth adding the config block to this btw since it's not just a matter of calling use e.g.
```lua
use {'kristijanhusak/orgmode.nvim', config = function()
  require('orgmode').setup{}
end
```

Reply:
Done! Thanks.